pages/faq.tsx

```diff
@@ -37,41 +32,8 @@ const questions: ReadonlyArray<{
             {' '}
             <Link href="/about"> for different actions</Link>
           </span>{' '}
-          - you get points for hosting a node and making transactions, and you
-          no longer get points directly for mining tokens. The other categories
-          for points remain the same.
-        </li>
-        <li>
-          Additionally, we have added a number of commands to the CLI to make
-          participating in the testnet simpler - these will help you set up your
-          graffiti and telemetry. Run <code>ironfish testnet</code> to set up
-          your node to be ready for testnet. Check out our{' '}
-          <span className="underline">
-            <Link href="https://ironfish.network/docs/onboarding/iron-fish-tutorial">
-              getting started
-            </Link>
-          </span>{' '}
-          for more details!
-        </li>
-      </div>
-    ),
-  },
-  {
-    question: `Why the point changes?`,
```

Comment:
we should explain this and keep this but empty out the explanation and write TODO